controls/SV-230342.rb

```diff
@@ -37,8 +37,8 @@
     $ sudo grep pam_faillock.so /etc/pam.d/password-auth
```

Comment:
**Note** paragraph in check text needs to be parameterized for `three` and `15`

Reply:
fixed: https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/commit/082f8a8681349c96bf8cb5667ede4874a8697b28